gitgud/__main__.py

```diff
@@ -148,10 +148,21 @@ def handle_start(self, args):
         python_exec = sys.executable.replace('\\', '/')  # Git uses unix-like path separators
 
         for git_hook_name, module_hook_name in all_hooks:
-            with open(os.path.join(self.file_operator.hooks_path, git_hook_name), 'w+') as hook_file:
+            path = os.path.join(self.file_operator.hooks_path, git_hook_name)
+            with open(path, 'w+') as hook_file:
                 hook_file.write('#!/bin/sh' + os.linesep)
-                hook_file.write('cat - | ' + python_exec + ' -m gitgud.hooks.' + module_hook_name + ' $1' +os.linesep)
-                hook_file.write('exit 0' + os.linesep)
+                hook_file.write('' + python_exec + ' -m gitgud.hooks.' + module_hook_name + os.linesep)
```

Comment:
Can you change that $1 to a $@? You removed it. Your current code breaks other git hooks. See #78